src/components/button/MenuButton.tsx

```diff
@@ -77,9 +54,9 @@ export const MenuButton = ({
   };
 
   return (
-    <Wrapper
+    <div
       ref={wrapperRef}
-      className={className}
+      className={clsx([styles.wrapper, className])}
```

Comment:
Does it need to be an array here 🤔 
```suggestion
      className={clsx(styles.wrapper, className)}
```